geom/opticks_raindrop.gdml

```diff
@@ -1,12 +1,23 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no" ?>
-<gdml xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="">
+<gdml_simple_extension xmlns:gdml_simple_extension="http://www.example.org"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema-instance"
+    xs:noNamespaceSchemaLocation="SimpleExtension.xsd">
```

Comment:
What is the point of switching to this new tag, gdml_simple_extension?

Reply:
Worked in CaTS, copied here. Not sure if has effect.